libs/pages/clusters/src/lib/feature/page-clusters-create-feature/page-clusters-create-feature.tsx

```diff
@@ -38,7 +38,7 @@ export const useClusterContainerCreateContext = () => {
   return clusterContainerCreateContext
 }
 
-export const steps = (cloudProvider?: CloudProviderEnum, clusterType?: string) => {
+export const steps = (cloudProvider = 'AWS', clusterType?: string) => {
```

Comment:
```suggestion
export const steps = (cloudProvider : CloudProviderEnum = 'AWS', clusterType?: string) => {
```
To avoid `cloudProvider` to be typed as `string`

Reply:
https://github.com/Qovery/console/pull/1105/commits/cf936ba724107effac122efe7a92c661a743c6d5